packages/cadl-ranch-specs/http/payload/content-negotiation/main.tsp

```diff
@@ -53,7 +53,7 @@ namespace DifferentBody {
   }
 
   model PngImageAsJson {
-    @header contentType: "image/jpeg";
+    @header contentType: "application/json";
```

Comment:
@timotheeguerin to confirm that this fix is correct

Reply:
The fix shall be right since accept of the op is `application/json`: https://github.com/Azure/cadl-ranch/blob/6fafb9a5bdced283985dfa8ed960791efb84d5fd/packages/cadl-ranch-specs/http/payload/content-negotiation/main.tsp#L64

Reply:
Yeah  that looks correct